test/integration-tests/src/test/resources/phase2-testgrid.yaml

```diff
@@ -0,0 +1,40 @@
+# TestGrid Test Configuration Parameters
+# TODO use dash as the separator instead of camelCase.
+version: '0.9'
+emailToList: "wso2testgridphase1@gmail.com,yasassri@wso2.com"
+infrastructureConfig:
+  iacProvider: CLOUDFORMATION
+  infrastructureProvider: AWS
+  containerOrchestrationEngine: None
+  parameters:
+    - JDK : ORACLE_JDK8
+  includes:
+     - CentOS
+     - mysql
+     - ORACLE_JDK8
+  provisioners:
+    - name: prod-wso2-ei-scenario-deployment
+      remoteRepository: "git@github.com:wso2/aws-ei.git"
```

Comment:
We need a specific commit hash instead of master branch, isn't it?

Reply:
ATM we are using the master with a specific product version